app/assets/javascripts/angular/controllers/public_relations.js

```diff
@@ -0,0 +1,65 @@
+angular.module('prApp', ['timer'])
+  .controller('prCtrl', ['$scope', '$http', '$interval', '$window', function($scope, $http, $interval, $window) {
+
+  $scope.focusedTab = 'income';
+
+  $scope.setTab = function(tab) {
+    $scope.focusedTab = tab;
+  }
+
+  $scope.nextRound = new moment();
+  $scope.news = [];
+
+  var apiCall = function() {
+    $http.get('/api/dashboard_data').then(
+      function successCallback(response) {
+        // Hide the connection error if it begins working again.
+        // See the Error callback for more information.
+        $('#connection-error').hide();
```

Comment:
Haven't incorporated this, not sure if it's something that mobile ought to have.